.editorconfig

```diff
@@ -6,7 +6,7 @@ root = true
 charset = utf-8
 end_of_line = lf
 insert_final_newline = true
-trim_trailing_whitespace = true
```

Comment:
I think we should remove this change in .editorconfig, other than that I think this is ready to go!